app/Http/Controllers/User/BlogController.php

```diff
@@ -23,7 +23,7 @@ public function defineUser() {
      */
     public function index()
     {
-        $blogs = Blog::all();
+        $blogs = Blog::where('user_id', '=', $this->defineUser()->id)->get();
```

Comment:
$canDelete = $this->defineUser()->can_delete === Blog::IS_TRUE?
or format code

Reply:
It can and i fixed